app/controllers/works_controller.rb

```diff
@@ -0,0 +1,83 @@
+class WorksController < ApplicationController
+  def index
+    works = Work.all
+    @works = works.sort_by {|work| -work.votes.count}
+    movies = Work.where(category: "movie")
+    @movies = movies.sort_by {|work| -work.votes.count}
+    books = Work.where(category: "book")
```

Comment:
This is definitely all functionality that ought to be in the model, but I still like it better here than if you had done it in the view.